src/DOM.ts

```diff
@@ -298,4 +298,33 @@ export class DOM {
     }
     return set;
   }
+
+
+  /**
+   * Heartland.DOM.addCertAlert
+   *
+   * Adds an alert letting the developer know they're in sandbox mode
+   *
+   * @param {Heartland.HPS} hps
+   * @param {string} elementid
+   */
+  public static addCertAlert() {
```

Comment:
Would you please move the function above the `private static` definitions above it?